microcosm_flask/tests/test_namespaces.py

```diff
@@ -120,6 +121,24 @@ def search_foo():
         assert_that(url, is_(equal_to("http://localhost/api/foo")))
 
 
+def test_operation_href_for_qs():
+    """
+    Operations can resolve themselves as fully expanded hrefs with
+    custom query string parameter.
+
+    """
+    graph = create_object_graph(name="example", testing=True)
+    ns = Namespace(subject="foo")
+
+    @graph.route(ns.collection_path, Operation.Search, ns)
+    def search_foo():
+        pass
+
+    with graph.app.test_request_context():
+        url = ns.href_for(Operation.Search, offset=0, limit=10, qs=dict(foo="bar"))
+        assert_that(url, matches_url("http://localhost/api/foo?offset=0&limit=10&foo=bar"))
```

Comment:
@jessemyers I realized when adding this unit-test that comparing URLs requires some canonicalization as otherwise the order of query strings is not guaranteed - I implemented as custom matcher using a third-party method I found. lmk if looks kosher to you, I imagine this matcher would be useful elsewhere also potentially